infra/gp-hashicorp-vault/templates/BackupTestJob.yaml

```diff
@@ -0,0 +1,116 @@
+{{- if .Values.backup.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: vault-backup-sa-auth-delegator
+  labels:
+        {{- include "gp-hashicorp-vault.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: 'system:auth-delegator'
+subjects:
+  - kind: ServiceAccount
+    name: vault-backup-sa
+    namespace: {{ .Release.Namespace }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vault-backup-tester-configmap
+  namespace: {{ .Release.Namespace }}
+  labels:
+      {{- include "gp-hashicorp-vault.labels" . | nindent 4 }}
+data:
+  config.hcl: |
+    disable_mlock = true
```

Comment:
das wirkt, wie wenn es ähnlich der config für die vault selbst in den values stehen?